README.md

```diff
@@ -9,17 +9,24 @@
 
 ## Getting Started
 
-This repository includes files with Ruby that can be used to decode a Morse code message:
+This repository includes files with Ruby that can be used to is getting information from the library about the University:
 
 - Please run the program in a IRB enviorement
-
-## Author
+- From your terminal you have to run the following command: `ruby main.rb`
+- After running this command you will get following kind of information:
+![screenshot](./Screenshot_2.png)
```

Comment:
- As said by the previous reviewer you should also add how to run test, you can write something like this :point_down: 
![image](https://user-images.githubusercontent.com/75126481/143508145-cd6ae86c-5cd4-4d72-8cd9-72534cee120a.png)

Reply:
Done ✔